<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TypingPractice</class>
 <widget class="QMainWindow" name="TypingPractice">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Typing Practice</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>581</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridMain">
     <item row="0" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>100</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="buttonStart">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonPause">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Pause</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonStop">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="buttonFile">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="fileLabel">
         <property name="font">
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>No File Selected</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonPause</sender>
   <signal>clicked()</signal>
   <receiver>TypingPractice</receiver>
   <slot>pausePractice()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonStop</sender>
   <signal>clicked()</signal>
   <receiver>TypingPractice</receiver>
   <slot>stopPractice()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonStart</sender>
   <signal>clicked()</signal>
   <receiver>TypingPractice</receiver>
   <slot>startPractice()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonFile</sender>
   <signal>clicked()</signal>
   <receiver>TypingPractice</receiver>
   <slot>getFileDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startPractice()</slot>
  <slot>pausePractice()</slot>
  <slot>stopPractice()</slot>
  <slot>getFileDialog()</slot>
 </slots>
</ui>
